setup.py

```diff
@@ -74,6 +74,7 @@ def local_file(name):
     zip_safe=False,
     extras_require=extras,
     install_requires=install_requires,
+    python_requires='==2.7,>=3.4',
```

Comment:
Certainly - I meant more that we should distinguish between source changes to make a release, and source changes to skip running tests.